vochain/scrutinizer/scrutinizer.go

```diff
@@ -97,26 +99,40 @@ func NewScrutinizer(dbPath string, app *vochain.BaseApplication) (*Scrutinizer,
 		return nil, err
 	}
 	startTime := time.Now()
-	envelopes, err := s.db.Count(&indexertypes.VoteReference{}, &badgerhold.Query{})
-	if err != nil {
-		return nil, fmt.Errorf("could not count the total envelopes: %w", err)
+
+	txCountStore := new(indexertypes.CountStore)
+	if err = s.db.FindOne(txCountStore,
+		badgerhold.Where(badgerhold.Key).Eq(indexertypes.Transactions)); err != nil {
+		log.Warnf("could not get the transaction count: %v", err)
```

Comment:
If the counters cannot be recovered from the KV, we should calculate them (using badgerhold.Count()) and store